playground/package.json

```diff
@@ -1,60 +1,48 @@
 {
-  "name": "playground",
+  "name": "playground2",
   "version": "1.0.0",
-  "description": "An interactive AMP playground.",
+  "description": "",
   "main": "index.js",
+  "sideEffects": [
+    "src/request-idle-callback",
+    "*.css"
+  ],
   "scripts": {
-    "start": "webpack-dev-server --config webpack.common.js",
-    "build": "rm -rf dist && webpack --config webpack.prod.js && cp -R static/* dist",
-    "test": "babel-node spec/run.js && eslint src/**/*.js",
-    "lint": "eslint src/**/*.js --fix"
+    "start": "webpack-dev-server --mode development --open",
+    "dev": "webpack --mode development",
+    "build": "rm -rf dist && webpack --mode production --optimize-minimize",
+    "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "author": "Sebastian Benz",
-  "license": "Apache-2.0",
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
Is there a reason for the changes here?

1. License to ISC
2. Removal of description
3. Name change `playground` -> `playground2`
4. From pinned dependencies to hat dependencies
5. Removal of tests and linting.

Reply:
see below :-)